fetch.bs

```diff
@@ -2409,10 +2409,10 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
  <li>
   <p>If <var>request</var>'s <a for=request>current URL</a>'s <a for=url>scheme</a> is
   "<code>ftp</code>" and <var>request</var>'s <a for=request>client</a>'s
-  <a for=environment>creation URL</a>'s <a for=url>scheme</a> is not "<code>ftp</code>",
-  and <var>request</var>'s <a for=request>reserved client</a> is either <code>null</code>
-  or an <a>environment</a> whose <a for=environment>target browsing context</a> is a
-  <a>nested browsing context</a>, set <var>response</var> to a <a>network error</a>.
+  <a for=environment>creation URL</a>'s <a for=url>scheme</a> is not "<code>ftp</code>", and
+  <var>request</var>'s <a for=request>reserved client</a> is either <code>null</code> or an
+  <a>environment</a> whose <a for=environment>target browsing context</a> is a <a>nested browsing
+  context</a>, then set <var>response</var> to a <a>network error</a>.
```

Comment:
Hrm. I thought y'all just didn't do newlines inside the tag, but that newlines inside the text were allowed. *shrug* I'll fix it up.